rust/lancedb/src/table.rs

```diff
@@ -1061,6 +1061,26 @@ impl NativeTable {
         }
     }
 
+    pub async fn get_index_type(&self, index_uuid: &str) -> Result<Option<String>> {
```

Comment:
Does this need to return Result if the result is always Ok?

Reply:
good point. There some other functions that are having similar issues. We can fix them all together.